app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,25 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def applicable_nations
+    all_nations = ['england', 'scotland', 'wales', 'northern_ireland']
+    inapplicabilities = item.nation_inapplicabilities
+    applies_to_all_nations = inapplicabilities.empty?
+
+    if applies_to_all_nations
+      nations = []
+    else
+      excluded_nations = inapplicabilities
+        .map(&:nation)
+        .map(&:name)
+        .map { |n| n.downcase.tr(' ', '_') }
```

Comment:
Feels wrong having to do this, but I couldn't find this already defined on the `Nation` model.